packages/sdk-ui/src/components/address-copy-button/address-copy-button.tsx

```diff
@@ -12,52 +18,65 @@ const sizeMap = {
 
 interface AddressCopyButtonProps {
   address: string;
+  showAddress?: boolean;
   size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+  containerStyle?: StyleProp<ViewStyle>;
   handleCopy?: (_: string) => void;
 }
 
 export const AddressCopyButton: React.FC<AddressCopyButtonProps> = ({
   address,
   size = 'sm',
+  showAddress = true,
+  containerStyle,
   handleCopy,
 }) => {
-  const [copied, setCopied] = useState(false);
   const [visible, setVisible] = useState(false);
 
   const handleCopied = useCallback(() => {
     handleCopy?.(address);
     if (!handleCopy) {
       console.warn(`No handleCopy function provided for address ${address}`);
     }
-    setCopied(true);
     setVisible(true);
     setTimeout(() => {
-      setCopied(false);
       setVisible(false);
     }, 1000);
   }, [address, handleCopy]);
 
   return (
-    <View style={styles.center}>
-      <IconButton
-        icon="content-copy"
-        size={sizeMap[size]}
-        onPress={handleCopied}
-      />
+    <View style={[styles.container, containerStyle]}>
+      <Pressable style={styles.pressable} onPress={handleCopied}>
+        {showAddress && (
```

Comment:
if `showAddress` is false we have a button with only the `conten-copy` icon here. Is it ok?